package.json

```diff
@@ -1,19 +1,24 @@
 {
   "name": "http-traceroute",
   "version": "1.6.1",
+  "main": "lib/trace.js",
   "description": "A command line tool for following and showing HTTP redirects for a given URL",
-  "bin": "index.js",
+  "bin": {
+    "http-traceroute": "bin/cli.js"
+  },
   "dependencies": {
     "chalk": "^1.1.3",
     "cookie-manager": "^0.0.16",
-    "normalize-url": "^1.4.1",
-    "spdy": "^3.3.2"
+    "normalize-url": "^1.9.0",
+    "spdy": "^3.4.4"
   },
   "devDependencies": {
-    "standard": "^6.0.8"
+    "mocha": "~3.2.0",
+    "standard": "^8.6.0"
   },
   "scripts": {
-    "test": "standard"
+    "lint": "standard lib && standard --global suite --global test",
+    "test": "mocha --ui tdd"
```

Comment:
I usually have `standard` as part of `npm test` to enhance the chance of users discovering their own linting errors. Otherwise contributors might just commit code that fails the linter and will not see that travis fails.